src/api/hooks/useGetAccountTokens.ts

```diff
@@ -59,7 +64,7 @@ export function useGetTokenData(tokenDataId?: string) {
         return undefined;
       }
       const response = await state.indexer_client?.getTokenData(tokenDataId);
-      return response?.current_token_datas_v2 ?? [];
+      return response?.current_token_datas_v2;
```

Comment:
Could be tricky changing the return type of a function that isn't explicitly typed. Looks like all usage of this function accounts for falsey though!